CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   **Bolt** Added `bolt admin login` to allow for logging in without an email provider setup. Automatically turns the user into an admin for immediate access to the developer dashboard.
 -   **Bolt** Fixed `bolt db migrate create`
 -   **Infra** Added `user-admin-set` service for creating an admin user
+-   **Infra** Added BetterUptime monitor for regions with pools
```

Comment:
Nit: "Better Uptime" has a space in it. Weird that the Terraform provider doesn't.

Reply:
Resolved in https://github.com/rivet-gg/rivet/commit/cf465364c533e4070a81090c3152b992a3fe154b

[![acceptit](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/WjlTemxb6oA4PgZFmj08/8488a0d6-7bb9-488a-90cd-060d546900aa/acceptit.gif)](https://app.graphite.dev/macros/rivet-gg)